src/components/FilmList/FilmList.tsx

```diff
@@ -0,0 +1,56 @@
+import "./style.scss";
+import { films } from "../../films";
+import { Film } from "../Film";
+import { FilmObj } from "../../interfaces";
+import ErrorBoundaries from "../ErrorBoundaries/ErrorBoundaries";
+
+interface FilmListProps {
+  genre: string;
+  searchFilm: string;
+  sort: string;
+}
+
+const FilmList: React.FC<FilmListProps> = ({ genre, searchFilm, sort }) => {
+  const filmes: any = films
+    .filter((el) => {
+      if (genre === "All") {
+        return true;
+      }
+      return el.genre.indexOf(genre) > -1;
+    })
+    .filter((el) => {
+      if (searchFilm === "") {
+        return true;
+      }
+
+      return el.name.toLowerCase().includes(searchFilm.toLowerCase());
+    })
+    .sort((a: FilmObj, b: FilmObj) => {
+      if (sort === "name") {
+        if (a.name < b.name) {
+          return -1;
+        }
+        if (a.name > b.name) {
+          return 1;
+        }
+        return 0;
+      } else {
+        return a.year - b.year;
+      }
+    })
+    .map((el: FilmObj, index: number) => {
+      return (
+        <ErrorBoundaries>
+          <Film film={el} key={index} />
```

Comment:
index is not a good key, for array render, its better to use `id` of the film or some unique number. Put `id` property into movies mock array